vmware/setup.sh

```diff
@@ -7,4 +7,9 @@ kubectl apply -f ./vmware/vcsim_deployment.yml
 while ! kubectl get deployment -n konveyor-forklift vcsim; do sleep 5; done
 kubectl wait deployment -n konveyor-forklift vcsim --for condition=Available=True --timeout=180s
 
-kubectl apply -f ./vmware/vsphere_provider.yml
+
+# workaround to https://github.com/kubevirt/kubevirt/issues/7078
+kubectl patch --type merge -p '{"spec": {"claimPropertySets": [{"accessModes": ["ReadWriteOnce"]}]}}' StorageProfile standard
```

Comment:
1. how about using the same mechanism for both ovirt and vsphere?
2. isn't it missing for openstack?

Reply:
1. sure, we can u

> @eslutsky you've addressed the comments nicely in the version that was merged but didn't leave much time for a follow up review.. added few comments/questions inside

sorry , though we are ready to merge, 
 i'll squeeze the follow-up fixes to this PR https://github.com/kubev2v/forkliftci/pull/62